applications/mobu/values.yaml

```diff
@@ -22,6 +22,11 @@ ingress:
   annotations: {}
 
 config:
+  # -- Which applications (tap, butler, etc.) are available in this environment.
+  # Notebooks can specify a `mobu.required_services` list in their metadata, and mobu will only run them if all services in that list are in this `available_services` list.
+  # See https://mobu.lsst.io/user_guide/in_repo_config.html#service-specific-notebooks
```

Comment:
Should wrap at 80 columns. The doc string uses Markdown, so the link should be a Markdown link (otherwise the whole URL will be added to the table and will probably break the formatting).